coq-principia-pl/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "1.0"
```

Comment:
This file should be at `released/packages/coq-principia-pl/coq-principia-pl.1.0.0/opam`.

Reply:
Thanks kindly for the corrections. I have moved the file to the suggested location.

Reply:
:+1: